docs/source/conf.py

```diff
@@ -93,6 +90,11 @@
     # The maximum depth of the table of contents tree. Set this to -1 to allow
     # unlimited depth.
     "navigation_depth": -1,
+    "show_prev_next": False,
+    # Navbar icon links
+    "navbar_start": ["navbar-logo"],
+    "switcher": {"json_url": "_static/switcher.json", "version_match": "23.9.x"},
```

Comment:
What does this `version_match` mean here, can we use "stable" instead?

Reply:
This was for the pydata-sphinx-theme's version matcher. I've instead opted to use the one provided ReadTheDocs.